api/lib/database/index.js

```diff
@@ -61,6 +63,8 @@ function database({ hasura_url, hasura_admin_key }) {
       })(schema),
       storeUser: storeUser({ hasura_url, hasura_admin_key })(schema),
       getUser: getUser({ hasura_url, hasura_admin_key })(schema),
+      getSession: getSession({ hasura_url, hasura_admin_key })(schema),
```

Comment:
i don't think this is self explanatory enough. maybe `getCurrentSession` or `getExistingSession`

Reply:
please take an example of @Bitcoinera and create a code change proposal for these things

Reply:
i am familiar with the option to do that - in this case i wasn't sure what you'd prefer.